api-spec/transactions-api.yaml

```diff
@@ -338,17 +335,28 @@ components:
               paymentToken: "paymentToken1"
               reason: "reason1"
               amount: 100
-              authToken: "authToken1"
             - rptId: "77777777777302012387654312385"
               paymentToken: "paymentToken2"
               reason: "reason2"
               amount: 100
-              authToken: "authToken2"
         status:
           $ref: "#/components/schemas/TransactionStatus"
+        amountTotal:
+          $ref: "#/components/schemas/AmountEuroCents"
+        feeTotal:
+          $ref: "#/components/schemas/AmountEuroCents"
+        origin:
+          description: cart origin
+          enum:
+            - IO
+            - CHECKOUT
+            - CHECKOUT_CART
+            - UNKNOWN
```

Comment:
why UNKNOWN?

Reply:
We have the same value in yaml file for document.
This value will be deleted, but, right now, we didn't know how to fill origin in transaction document, or better, we know, but we didn't yet implement it.

Reply:
good point,  next refinement :-)